Steps/Steps.module.less

```diff
@@ -47,10 +47,10 @@
 
 .applySkins({
 	.step {
-		&.dots {
-			color: @sand-steps-step-focused-dot-color;
+		&.highlightCurrent {
```

Comment:
Please rename this to `highlightCurrentOnly`.

Reply:
It's fixed.